Mta 6787 (#5916)
scaleway/docs-content: pages/gpu/how-to/use-preinstalled-env.mdx

|        OS image        | Image release type |    Preinstalled on image                                        | Working environment                            |
|:----------------------:|:------------------:|:---------------------------------------------------------------:|------------------------------------------------|
| Ubuntu Focal GPU OS 12 | Legacy             | Nvidia drivers, Nvidia Docker environment (launch Docker container to access working environment) | Pipenv virtual environment accessed via Docker |

> | Ubuntu Noble GPU OS 13 (Nvidia) | Latest    | Nvidia drivers, Nvidia Docker environment (launch Docker container to access working environment) | Pipenv virtual environment accessed via Docker |

Reviewer: ```suggestion
| Ubuntu Noble GPU OS 13 (NVIDIA) | Latest    | NVIDIA drivers, Nvidia Docker environment (launch Docker container to access working environment) | Pipenv virtual environment accessed via Docker |
```

The Ubuntu Noble GPU OS 13 (Nvidia) contains the latest version of the [Nvidia open-source drivers](https://developer.nvidia.com/blog/nvidia-releases-open-source-gpu-kernel-modules/#:~:text=This%20release%20is%20a%20significant,modules%20increase%20ease%20of%20use) required for Blackwell GPU architecture.

> Using the latest Ubuntu Noble GPU OS13 image gives you a minimal OS installation on which you can [launch](/gpu/how-to/use-gpu-with-docker/) one of our ready-made [Docker images](/gpu/reference-content/docker-images/). This gives you access to a preinstalled Python environment managed with pipenv. A number of useful AI core packages and tools are installed, including `scipy`, `numpy`, `scikit-learn`, `jupyter`, `tensorflow`, and the Scaleway SDK. Depending on the [Docker image you choose](/gpu/reference-content/docker-images/), other packages and tools will also be preinstalled, providing a convenient framework environment for you so that you can begin work immediately.

Reviewer: ```suggestion
Using the latest Ubuntu Noble GPU OS13 image gives you a minimal OS installation on which you can [launch](/gpu/how-to/use-gpu-with-docker/) one of our ready-made [Docker images](/gpu/reference-content/docker-images/). This gives you access to a preinstalled Python environment managed with pipenv. Several useful AI core packages and tools are installed, including `scipy`, `numpy`, `scikit-learn`, `jupyter`, `tensorflow`, and the Scaleway SDK. Depending on the [Docker image you choose](/gpu/reference-content/docker-images/), other packages and tools will also be preinstalled, providing a convenient framework for you to begin work immediately.
```
